code/modules/guardian/abilities/major/gravity.dm

```diff
@@ -0,0 +1,52 @@
+/datum/guardian_ability/major/gravity
+	name = "Gravity"
+	desc = "The guardian's punches apply heavy gravity to whatever it punches."
+	cost = 4
```

Comment:
should probably lower this

Reply:
this is literally broken in so many ways and is only balanced by its cost that you need to spec into potential and damage to be useful